lib/components/viewers/nearby/vehicle-rent.tsx

```diff
@@ -119,6 +120,18 @@ const Vehicle = ({
         )}
         {fromToSlot}
       </CardBody>
+      {vehicle.additionalCount > 0 && (
+        <CardFooter>
+          <span>
+            <FormattedMessage
+              id="components.NearbyView.additionalVehicles"
```

Comment:
Was this layout more advantageous than placing this text underneath the distance label (see illustration)?
<img width="383" alt="image" src="https://github.com/user-attachments/assets/f9f98adb-1b2c-4eaf-9dd1-8e6c2add9189" />

Reply:
I thought about this but it gets busy especially in the case of a 2-line title